src/csl/rendering/mod.rs

```diff
@@ -85,6 +85,11 @@ impl RenderCsl for citationberg::Text {
                 {
                     render_typed_num(num.as_ref(), NumberForm::default(), var, None, ctx);
                 }
+                NumberVariableResult::Regular(n)
+                    if matches!(var, NumberVariable::Page) =>
+                {
+                    ctx.push_str(&n.to_str().replace("-", "–"))
```

Comment:
Does the CSL standard hard-code the en-dash here?

Reply:
Not really. Honestly, this is a quick fix so we can handle these cases correctly (for the most part). I will work rework the page ranges in general. They should be able to handle more than numbers. I should get around to that some time next week.

Reply:
Got it